core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackDtoService.java

```diff
@@ -233,18 +227,13 @@ public StackView getStackViewByCrn(String crn) {
         return stackViewDelegate.orElseThrow(NotFoundException.notFound("Stack by crn", crn));
     }
 
-    public Optional<StackView> getStackViewByCrnOpt(String crn) {
-        Optional<StackViewDelegate> stackViewDelegate = stackDtoRepository.findByCrn(crn);
-        return Optional.ofNullable(stackViewDelegate.orElseThrow(null));
-    }
-
     public StackView getStackViewById(Long id) {
         return stackDtoRepository.findById(id).orElseThrow(NotFoundException.notFound("Stack", id));
     }
 
-    public Optional<StackView> getStackViewByIdOpt(Long id) {
+    public StackView getStackViewByIdOpt(Long id) {
```

Comment:
Nit: perhaps should be renamed to `getStackViewById()` as it does not result in an `Optional` anymore.

Reply:
thanks, fixed